src/rdkafka_admin.c

```diff
@@ -5244,6 +5245,18 @@ void rd_kafka_AlterConsumerGroupOffsets(
                 }
```

Comment:
In the line 5231, we should also add a NULL check `alter_grpoffsets[0]->partitions == NULL || alter_grpoffsets[0]->partitions->cnt == 0`

Reply:
Good to know.